package.json

```diff
@@ -9,7 +9,7 @@
     "lint:js": "eslint --ext=.js,.mjs .",
     "format:js": "npm run lint:js -- --fix",
     "lint:scss": "stylelint _sass",
-    "check:html": "bundle exec jekyll build && node _tasks/check-html.mjs",
+    "check:html": "bundle exec jekyll build && html-validate _site/index.html",
```

Comment:
would "lint:html" be more consistent with the above scripts? Does it have to be "check"?

Reply:
SGTM, let's rename it to `lint:html`. @osBins also remember to change in the CI script.